sylt-parser/src/statement.rs

```diff
@@ -19,7 +28,8 @@ pub enum StatementKind {
     /// `use <file> as <alias>`.
     /// `use <folder>/ as <alias>`.
     Use {
-        ident: Identifier,
+        name: Identifier,
+        alias: UseIdentifier,
```

Comment:
These names are hard.

Reply:
I would probably do path and name instead of name and alias